fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java

```diff
@@ -164,14 +167,17 @@ private void writeUnlock() {
 
     // NOTICE: call 'addTableToGroup()' will not modify 'group2BackendsPerBucketSeq'
     // 'group2BackendsPerBucketSeq' need to be set manually before or after, if necessary.
-    public GroupId addTableToGroup(long dbId, OlapTable tbl, String groupName, GroupId assignedGroupId) {
+    public GroupId addTableToGroup(long dbId, OlapTable tbl, String groupName, GroupId assignedGroupId, boolean isReplay)
```

Comment:
Need to check that local table and lake table are in the same colocate group?

Reply:
done